azure-pipelines.yml

```diff
@@ -20,56 +28,73 @@ stages:
         parameters:
           zone: 'demo'
           build: $(Build.BuildNumber)
           agentPool: ${{ variables.agentPool }}
-          terraformVersion: ${{ variables.terraformVersion }}
+          backendServiceArm: 'Reform-CFT-Mgmt'
+          backendAzureRmResourceGroupName: 'reform-cft-mgmt'
+          backendAzureRmStorageAccountName: 'reformcftmgmt'
 
+          
       - template: pipeline-templates/terraform-plan.yaml
         parameters:
           zone: 'dev'
           build: $(Build.BuildNumber)
           agentPool: ${{ variables.agentPool }}
-          terraformVersion: ${{ variables.terraformVersion }}
+          backendServiceArm: 'Reform-CFT-Mgmt'
+          backendAzureRmResourceGroupName: 'reform-cft-mgmt'
+          backendAzureRmStorageAccountName: 'reformcftmgmt'
           
       - template: pipeline-templates/terraform-plan.yaml
         parameters:
           zone: 'ithc'
           build: $(Build.BuildNumber)
           agentPool: ${{ variables.agentPool }}
-          terraformVersion: ${{ variables.terraformVersion }}
+          backendServiceArm: 'Reform-CFT-Mgmt'
+          backendAzureRmResourceGroupName: 'reform-cft-mgmt'
+          backendAzureRmStorageAccountName: 'reformcftmgmt'
 
       - template: pipeline-templates/terraform-plan.yaml
         parameters:
           zone: 'test'
           build: $(Build.BuildNumber)
           agentPool: ${{ variables.agentPool }}
-          terraformVersion: ${{ variables.terraformVersion }}
+          backendServiceArm: 'Reform-CFT-Mgmt'
+          backendAzureRmResourceGroupName: 'reform-cft-mgmt'
+          backendAzureRmStorageAccountName: 'reformcftmgmt'
 
       - template: pipeline-templates/terraform-plan.yaml
         parameters:
           zone: 'staging'
           build: $(Build.BuildNumber)
           agentPool: ${{ variables.agentPool }}
-          terraformVersion: ${{ variables.terraformVersion }}
+          backendServiceArm: 'Reform-CFT-Mgmt'
+          backendAzureRmResourceGroupName: 'reform-cft-mgmt'
+          backendAzureRmStorageAccountName: 'reformcftmgmt'
 
       - template: pipeline-templates/terraform-plan.yaml
         parameters:
           zone: 'nle'
           build: $(Build.BuildNumber)
           agentPool: ${{ variables.agentPool }}
-          terraformVersion: ${{ variables.terraformVersion }}
+          backendServiceArm: 'Reform-CFT-Mgmt'
+          backendAzureRmResourceGroupName: 'reform-cft-mgmt'
+          backendAzureRmStorageAccountName: 'reformcftmgmt'
 
       - template: pipeline-templates/terraform-plan.yaml
         parameters:
           zone: 'prod'
           build: $(Build.BuildNumber)
           agentPool: ${{ variables.agentPool }}
-          terraformVersion: ${{ variables.terraformVersion }}
+          backendServiceArm: 'Reform-CFT-Mgmt'
+          backendAzureRmResourceGroupName: 'reform-cft-mgmt'
+          backendAzureRmStorageAccountName: 'reformcftmgmt'
```

Comment:
I probably wouldn't bother duplicating these all over the place because it's always the same, you could make them parameters that are defaulted in the scripts or just hardcode the 2 usages, in private dns 2 subscriptions are used i think (sandbox is in  a different one)

Reply:
as in make a variable and refer to the variable like the line above (agentPool)?